pip_review/__main__.py

```diff
@@ -244,55 +136,59 @@ def confirm(question):
     return answer == 'y'
 
 
+def get_outdated_packages(local=False, pre_release=False, user=False):
+    command = ['pip', 'list', '--outdated', '--disable-pip-version-check']
```

Comment:
It will break the output for people using older versions of pip :( as it will throw a warning message to update to the new version of pip; that arguments disables it...